src/realtime-gtfs/router.js

```diff
@@ -1,135 +1,183 @@
 import express from 'express';
+import { checkSchema, validationResult } from 'express-validator';
 import realtimeClient from './client.js';
 import cache from '../lib/cache.js';
 import logger from '../lib/logger.js';
 import {
-  GTFS_REALTIME_ALERTS_URL,
   GTFS_REALTIME_VEHICLE_POSITIONS_URL,
   GTFS_REALTIME_TRIP_UPDATES_URL,
 } from '../config.js';
+import GtfsRealtimeBindings from 'gtfs-realtime-bindings';
 
 const vehiclePositionsUrl = GTFS_REALTIME_VEHICLE_POSITIONS_URL;
-const serviceAlertsUrl = GTFS_REALTIME_ALERTS_URL;
 const tripUpdatesUrl = GTFS_REALTIME_TRIP_UPDATES_URL;
 
 const router = express.Router();
 
-async function vehiclePositionsCb (req, res) {
-  if (!vehiclePositionsUrl) {
-    logger.info('env var GTFS_REALTIME_VEHICLE_POSITIONS_URL not found');
-    res.sendStatus(404);
-    res.end();
-    return;
+const validateQueryParams = () => checkSchema({
+  'trip-id': {
+    optional: true,
+    trim: true,
+    isLength: {
+      options: { 
+        min: 3,
+        max: 64
```

Comment:
The min is because that is as small as it can be since the format is CHAR colon CHAR. The max is based on checking the lengths of all the current trip and route IDs and then rounding up. I think the longest was 43.